packages/frontend/components/Forms/TextInput.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './Form.module.scss'
+
+export default function TextInput({ name, label, error }) {
+  return (
+    <div className={styles.textInput}>
+      <label htmlFor={name}>{label}</label>
+      <input type="text" name={name} />
+      {error && <div className={styles.error_msg}>{error}</div>}
+    </div>
+  )
+}
+
+TextInput.propTypes = {
+  name: PropTypes.string,
+  label: PropTypes.string,
+  error: PropTypes.string,
+}
```

Comment:
Is there a reason you're doing this with proptypes instead of a typescript interface? (I'll just leave one comment for this instead of one on each component)

Reply:
I'm not sure what's the best practice to give type for component props. Do you have code example of using interface as a type for props?

Reply:
Oh, I got it, changed it to interface